db.go

```diff
@@ -841,6 +841,17 @@ func (db *DB) Close(options ...CloseOption) error {
 	}
 	level.Info(db.logger).Log("msg", "closed all tables")
 
+	if !shouldPersist && db.columnStore.snapshotTriggerSize != 0 {
+		db.snapshot(context.Background(), false, func() {
```

Comment:
I think it would be nice to info log the duration this took

Reply:
Done.